src/test/regress/pg_regress_multi.pl

```diff
@@ -289,6 +289,7 @@ sub revert_replace_postgres
 # not required, and we don't necessarily have access to the default directory
 push(@pgOptions, '-c', "unix_socket_directories=");
 push(@pgOptions, '-c', "fsync=off");
+push(@pgOptions, '-c', "extra_float_digits=0");
```

Comment:
This is already the default, why do we set it again?

Reply:
https://www.postgresql.org/docs/12/release-12.html

> Improve performance by changing the default number of trailing digits output for REAL and DOUBLE PRECISION values (Andrew Gierth)
> Previously, float values were output rounded to 6 or 15 decimals by default. Now, only the number of digits required to preserve the exact binary value is output. The previous behavior can be restored by setting extra_float_digits to zero.